<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoblinCoding's XML Mill</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/resources/goblinicon.png</normaloff>:/resources/goblinicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="DomTreeView" name="treeView">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
      <widget class="ElementEditTableWidget" name="tableWidget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="expandAllCheckBox">
        <property name="text">
         <string>Expand All</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuAddItems">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Add elements and attributes to the active profile.</string>
     </property>
     <property name="whatsThis">
      <string>Add elements and attributes to the active profile.</string>
     </property>
     <property name="title">
      <string>Edit &amp;Profile</string>
     </property>
    </widget>
    <addaction name="menuAddItems"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionCloseFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionRememberWindowGeometry"/>
    <addaction name="actionShowHelpButtons"/>
    <addaction name="actionUseDarkTheme"/>
    <addaction name="separator"/>
    <addaction name="actionForgetPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpContents"/>
    <addaction name="actionVisitOfficialSite"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuAdvanced"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionHelpContents"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="toolTip">
    <string>Open an XML file.</string>
   </property>
   <property name="whatsThis">
    <string>Open an XML file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/save.png</normaloff>:/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save File</string>
   </property>
   <property name="whatsThis">
    <string>Save XML file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save File &amp;As</string>
   </property>
   <property name="whatsThis">
    <string>Sae XML file as.</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/new.png</normaloff>:/resources/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New File</string>
   </property>
   <property name="toolTip">
    <string>Create new XML file.</string>
   </property>
   <property name="whatsThis">
    <string>Create new XML file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/close.png</normaloff>:/resources/close.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="whatsThis">
    <string>Close the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionForgetPreferences">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/clear.png</normaloff>:/resources/clear.png</iconset>
   </property>
   <property name="text">
    <string>Forget Message Preferences</string>
   </property>
   <property name="whatsThis">
    <string>Forget all saved preferences regarding messages/warnings/queries.</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/resources/helpicon.png</normaloff>:/resources/helpicon.png</iconset>
   </property>
   <property name="text">
    <string>Help &amp;Contents</string>
   </property>
   <property name="whatsThis">
    <string>Displays the main &quot;Help&quot; contents.</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionVisitOfficialSite">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/goblin.png</normaloff>:/resources/goblin.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Visit Official Website</string>
   </property>
   <property name="whatsThis">
    <string>Visit the official website (will open in your browser).</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close File</string>
   </property>
   <property name="toolTip">
    <string>Close the current file without saving.</string>
   </property>
   <property name="whatsThis">
    <string>Close the current file without saving.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionRememberWindowGeometry">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remember Window Geometry</string>
   </property>
   <property name="toolTip">
    <string>Start XML Mill in the same place and shape next time</string>
   </property>
   <property name="whatsThis">
    <string>Start XML Mill in the same place and shape next time</string>
   </property>
  </action>
  <action name="actionUseDarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Use Dark Theme</string>
   </property>
  </action>
  <action name="actionShowHelpButtons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Help Buttons</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DomTreeView</class>
   <extends>QTreeView</extends>
   <header>view/domtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>ElementEditTableWidget</class>
   <extends>QTableWidget</extends>
   <header>widgets/elementedittablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
